ci/Dockerfile.fedora

```diff
@@ -40,11 +40,11 @@ RUN dnf makecache && dnf install -y \
     shtool \
     unzip \
     wget \
+    which \
     zlib-devel
 
 # Install the Cloud Bigtable emulator and the Cloud Bigtable command-line
 # client.  They are used in the integration tests.
-ARG GOPATH=/var/tmp/build/cbt
-WORKDIR ${GOPATH}
-RUN go get cloud.google.com/go/bigtable/cmd/cbt \
-    cloud.google.com/go/bigtable/cmd/emulator
+WORKDIR /var/tmp/install/cbt-components
+RUN curl https://sdk.cloud.google.com | bash -s -- --disable-prompts --install-dir=/usr/local
+RUN /usr/local/google-cloud-sdk/bin/gcloud --quiet components install cbt bigtable
```

Comment:
https://cloud.google.com/bigtable/docs/emulator says you need to run

```
gcloud components update beta
```

whereas this is running

```
gcloud components install cbt bigtable
```

Are they equivalent?

Reply:
I do not believe they are equivalent.  After running:

```console
$ gcloud components update beta
```

the Google Cloud Bigtable Emulator is not installed.  If you try to start it interactively you get:

```console
$ /usr/local/google-cloud-sdk/bin/gcloud beta emulators bigtable start
You need the [bigtable] component to use the Google Cloud Bigtable 
emulator.
...[snip snip snip]
To install them, re-run the command and choose 'yes' at the installation
prompt, or run:
  $ gcloud components install bigtable
```

I think the Dockerfiles are correct, maybe we should update the Cloud Bigtable site and save the users a step?